projects/plugins/jetpack/_inc/client/earn/ads.jsx

```diff
@@ -356,6 +359,19 @@ export const Ads = withModuleSettingsFormHelpers(
 						) }
 					</SettingsGroup>
 					{ ! isSubDirSite && this.renderAdsTxtSection() }
+					<SettingsGroup>
```

Comment:
I think we should move this toggle above Ads.txt so it's grouped with the other privacy controls.

Reply:
That works too. I positioned it below from UI perspective because it doesn't have the help info icon on the right. So all the text with the more info icon are grouped above